build/Build.cs

```diff
@@ -262,9 +283,15 @@ public Build()
                     foreach (var flavour in calamariFlavours)
                     {
                         Log.Verbose($"Compressing Calamari flavour {PublishDirectory}/{flavour.Name}");
-                        CompressionTasks.CompressZip(PublishDirectory / flavour.Name, $"{ArtifactsDirectory / flavour.Name}.zip");
+                        var compressionSource = PublishDirectory / flavour.Name;
+                        if (!Directory.Exists(compressionSource))
```

Comment:
I think this should error out If the directory doesn't exist and we're not copying the flavour across, am I missing something here? If not should we just throw an exception or let the CompressZip method throw an exception?

Reply:
I added this to suppress a local error. For flavours like this that only has netfx (windows only) targets, when building it on Mac, since every package was simply skipped, the output directory ends up not being created. So at the compression step it will give an error saying that the directory does not exist.

Reply:
Ahh that makes sense, I'm assuming building on mac skipped these packages previously aswell?

Reply:
Yes, window-only builds were skipped on Mac. In CI they should work fine.

Reply:
All good by me then 🙂